harper-core/src/spell/full_dictionary.rs

```diff
@@ -62,7 +67,7 @@ impl FullDictionary {
 
     /// Create a dictionary from the curated dictionary included
     /// in the Harper binary.
-    pub fn curated() -> Lrc<Self> {
+    pub(super) fn curated() -> Arc<Self> {
```

Comment:
Rather than making it private to the module, add a comment warning people which situations `FstDictionary` is a better choice.